src/Action/ConvertPaymentAction.php

```diff
@@ -15,6 +15,7 @@
 
 use libphonenumber\PhoneNumberUtil as PhoneNumberUtil;
 use libphonenumber\PhoneNumberFormat as PhoneNumberFormat;
+use Sylius\Component\Customer\Model\CustomerInterface;
```

Comment:
it is better to use CustomerInterface from the core instead : Sylius\Component\Core\Model\CustomerInterface